packages/storybook/src/generators/configuration/schema.json

```diff
@@ -18,15 +18,22 @@
       "x-dropdown": "projects",
       "x-priority": "important"
     },
+    "interactionTests": {
+      "type": "boolean",
+      "description": "Set up Storybook interaction tests.",
+      "x-prompt": "Do you want to set up Storybook Interaction Tests?",
```

Comment:
Are we calling it `Interaction Test` or `Interaction Testing`? If it's used as a noun (implied by the capitalized words) we should be consistent. I think it's better to say `Storybook interaction tests`.